glide.yaml

```diff
@@ -93,7 +93,6 @@ import:
 - package: k8s.io/client-go
   version: v2.0.0
 - package: github.com/gambol99/go-marathon
-  version: d672c6fbb499596869d95146a26e7d0746c06c54
```

Comment:
Sure, understood. FWIW, a number of glide problems can be resolved by clearing the cache up-front (i.e., running `glide cc`); but I know from experience that it doesn't always help.

I updated the YML file and hash. Hope it's okay now.